src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public abstract class Figure {
+    public static final double COEFFICIENT = 0.5;

```

Comment:
All magic numbers in your code should be constants. If the COEFFICIENT is specific to a certain figure, it should be defined in that figure's class, not in the abstract Figure class.